content/applications/hr/payroll.rst

```diff
@@ -311,15 +314,25 @@ The required fields for an advantage are:
   :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
 - :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
   advantage applies to.
+- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+
+.. image:: payroll/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
 
```

Comment:
It would be nice to have some background on the options we are picking here. These forms can be overwhelming. Remember to add sections.

Reply:
I thought the explanation of what an advantage was, and why you'd add one was enough and clear. I did add the sections and included the non-required information to add as well, so that part is taken care of!